### ISSUE TITLE
Images on the archive page are missing alt text

### ISSUE THREAD

**Issue by kaitlinnewson** [NateWr/bootstrap3#141]
One of our journals has noticed that when a cover image has alt text, it is displayed on the issue page, but is missing on the archive page.

For example, this issue cover image has alt text: https://journal.lib.uoguelph.ca/index.php/perj/issue/view/380

But it is missing in the archive page: https://journal.lib.uoguelph.ca/index.php/perj/issue/archive

From what I can tell, it seems to be missing in `templates/frontend/objects/issue_summary.tpl` - if this looks correct, I'm happy to submit a PR!

**Comment by NateWr** [comment]
Yeah, a PR would be great. Can you update the `issue_summary.tpl` and if needed the `issue_toc.tpl` to include the `null` default, as shown here:

https://github.com/pkp/ojs/blob/master/templates/frontend/objects/issue_summary.tpl#L22

That's a recommendation from our accessibility audit that we want to roll out to all of the themes.

**Comment by kaitlinnewson** [comment]
When I add `default:'null'`, this results in `alt="null"` as the output; is that the expected result? Or should the output be `alt=""`? Having `alt="null"` read out to a screenreader user seems like it would contribute to a negative user experience.

**Comment by NateWr** [comment]
Yes, that's correct. I only recently learned myself, but using `alt="null"` will tell a screenreader to skip the element entirely.

**Comment by NateWr** [comment]
You're right! Thanks for setting me straight on this one. You can use smarty's `default` syntax with an empty string like this: https://github.com/pkp/pkp-lib/commit/e9902f9a2edbc12034296f03fabc550315f6f3c1#diff-9cb3646f5335fd2b497d4a0fcb640f91R17

**Comment by NateWr** [comment]
Thanks!
